pkg/blueprint/repository_customizations.go

```diff
@@ -76,6 +80,17 @@ func (rc *RepositoryCustomization) getFilename() string {
 	return rc.Filename
 }
 
+func RepoCustomizationsFilterEnabledDuringBuild(repos []RepositoryCustomization) []RepositoryCustomization {
```

Comment:
Would it make sense to return `([]rpmmd.RepoConfig, error)` from the function (basically just calling `.customRepoToRepoConfig()` on the repo)? Calling `RepoCustomizationsToRepoConfigAndGPGKeyFiles()` seems to do much more than just conversion and the resulting `[]*fsnode.File` is always discarded for these repos.

Reply:
Thanks, that is a nice one - much simpler this way